contracts/bonding/BondingManager.sol

```diff
@@ -566,12 +555,68 @@ contract BondingManager is IBondingManager, Manager {
         }
     }
 
-    /*
-     * @dev Return number of minted tokens for a reward call
-     */
-    function mintedTokensPerReward(address _transcoder) public constant returns (uint256) {
-        uint256 transcoderActiveStake = activeTranscoders[activeTranscoderPositions[_transcoder]].key;
-        return initialTokenSupply.mul(initialYearlyInflation).div(100).div(roundsManager().roundsPerYear()).mul(transcoderActiveStake).div(totalActiveTranscoderStake);
+    // Transcoder getters
+
+    function getTranscoderDelegatorWithdrawRound(address _transcoder) public constant returns (uint256) {
```

Comment:
We need these getters to individually retrieve values for a transcoder or delegator struct (the proxy contract cannot return tuples at the moment)

Reply:
Not sure, I'll make a note to test this out. Since constant getters are using the eth_call RPC request for a node maybe we can batch together the RPC requests to make it faster